stable/stackdriver-exporter/README.md

```diff
@@ -0,0 +1,103 @@
+# Stackdriver Exporter
+
+Prometheus exporter for Stackdriver, allowing for Google Cloud metrics.  You
+must have appropriate IAM permissions for this to exporter to work.  If you
```

Comment:
@apenney Out of curiosity, do need to specify a ServiceAccount name to the app? I haven't installed this, but wondering how these roles are recognized by the chart.

Reply:
I wasn't sure how to handle that.  Generally GKE requires "monitor" oauth scope to be enabled (or at least tells you to enable it in the instructions) so almost all clusters should be able to grab these metrics for themselves at least.

The use case of "cross project gathering" I'm still not 100% on!